CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.0.0-rc.7 (2020-04-28)
+
+## Breaking changes
+
+- Changed the name of the entry point file from `./src/app/public/index.ts` to `/src/app/public/public_api.ts` when building SKY UX Libraries. [#248](https://github.com/blackbaud/skyux-sdk-builder/pull/248)
```

Comment:
What do you think about expanding this to include why we made the change? Just to indicate why this warranted a breaking change. I'm thinking something like:

"Changed the name of the entry point file for building SKY UX libraries from `./src/app/public/index.ts` to `./src/app/public/public_api.ts`. This follows Angular guidance against `index.ts` barrel files in libraries and mimics Angular CLI requirements."